File: angular/src/app/Shared/Models/crew.ts
import { Pilot } from "./pilot";
import { Stewardess } from "./stewardess";

export class Crew {
    constructor(
        public id?: number,
        public pilot?: Pilot,
        public stewardesses?: Stewardess[]) {
            
        this.pilot = new Pilot();

    }
}

File: Airport/Shared/DTO/Crew.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Shared.DTO
{
    public class Crew
    {
        public int Id { get; set; }
        [Required]
        public Pilot Pilot { get; set; }
        [Required]
        public List<Stewardess> Stewardesses { get; set; }
    }
}
